config/features.yml

```diff
@@ -202,6 +202,15 @@ features:
   decision_review_hlr_status_update_throttling:
     actor_type: user
     description: Throttles requests to CMP when updating HLR statuses
+  decision_review_hlr_pii_expunge_enabled:
+    actor_type: user
+    description: Enables the higher level review PII expunge job
+  decision_review_nod_pii_expunge_enabled:
+    actor_type: user
+    description: Enables the notice of disagreement PII expunge job
+  decision_review_sc_pii_expunge_enabled:
+    actor_type: user
+    description: Enables the supplemental claim PII expunge job
```

Comment:
I'm not sure what the default value for a Flipper flag is -- is there someplace I should set it to true to match the old settings?

Reply:
It looks like, by default, flipper flags are off in production:
[code](https://github.com/department-of-veterans-affairs/vets-api/blob/6612b8fbe25ff15d4ef7c007fafad8854b7478d5/config/features.yml#L4):
# Features are enabled by default in the test environment and disabled by default in other environments.
# To default a feature to enabled in development, set the `enable_in_development` key to true.

Reply:
Thanks! I've updated this to enable them all in development - I imagine we'll need to turn them on in production before deploying this, too, although I don't have access yet